app/views/provider/admin/cms/pages/_form.html.erb

```diff
@@ -5,7 +5,8 @@
   <%= f.input :title, :autofocus => true %>
 
   <%= f.input :section, :as => :section, :default => current_account.sections.root, :paths => true, input_html: {class: 'cms-section-picker'} %>
-  <%= f.input :path, :hint => 'Does not depend on a selected section.', input_html: {class: 'cms-path-autocomplete'} %>
+  <%= f.input :path, :hint => 'Does not depend on a selected section.', input_html: {class: 'cms-path-autocomplete', name: 'cms_template[path]'} %>
```

Comment:
Why is this change applied to a single input? :upside_down_face: 

We need to find the root cause of why all the attributes are supplied under `cms_page` and fix the root cause.

Reply:
The reason I did this is because path  is coming under cms_page and not getting updated in db, for other attributes I have not change because they were actually getting updated correctly without any issue. The problem was with path and when things created or updated correctly that time too all attributes comes under cms_page and only path with cms_template.

Reply:
This issue happens for all fields, not only the path :thinking:

Reply:
i was not able to reproduce for any other field

Reply:
Exactly the same is reproduced with Title. The others are harder to reproduce because they are drop-downs.